extra-cpp-bindings/src/lib.rs

```diff
@@ -132,6 +132,15 @@ mod ffi {
         pub token_type: String,
     }
 
+    /// Token holder detail from BlockScout API
+    #[derive(Serialize, Deserialize, Debug, PartialEq, Eq)]
```

Comment:
PartialEq is necessary?

Reply:
yes, just keep the same as other types.

Reply:
I think it's needed for tests / assert equals?